### DISCUSSION TITLE
Create3 BatteryState.design_capacity going down?

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#248] [Questions] [answered]
Four days ago when my Create3 was new, the BatteryState.design_capacity was 2.052 AHrs.

Today it is 2.046 AHrs.  (I'm really going to miss those 6mAHrs...might have powered my 13W bot for another 0.2 seconds)

I understand BatteryState.capacity will go down as the battery ages, but I don't understand the "design capacity" changing.

**Comment by alsora** [comment, marked as answer]
Currently, `design_capacity` and `capacity` are populated using the same value (i.e. the current battery capacity)

We should probably change `design_capacity` to be a fixed value.

**Comment by slowrunner** [comment]
closing - answered
